app/frontend/stylesheets/pages/_home.scss

```diff
@@ -10,4 +10,13 @@
     flex-direction: column;
     margin: 50px 0;
   }
+
+  .cal-event-cards {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 50px;
+    margin-bottom: 50px;
+  }
+
```

Comment:
Can you move this inside the #planning proprety please.
There's also a trailing empty line that you can remove at line 21